tutorials/scripting/nodes_and_scene_instances.rst

```diff
@@ -0,0 +1,205 @@
+          practice is to avoid doing that though not to break encapsulation and
+          keep your code organized. You can also start the path with a forward
+          slash to make it absolute, in which case your topmost node would be
+          "/root", the application's predefined root viewport.
+
+Syntactic sugar
+~~~~~~~~~~~~~~~
+
+You can use two shorthands to shorten your code in GDScript: putting the
+``onready`` keyword before a member variable makes it initialize right before
+the ``_ready()`` callback.
+
+.. code-block:: gdscript
+
+    onready var sprite = get_node("Sprite")
+
+There is also a short notation for ``get_node()``: the dollar sign, "$". You
+place it before the name or path of the node you want to get.
+
+.. code-block:: gdscript
+
+    onready var sprite = $Sprite
+    onready var tween = $ShieldBar/Tween
+
+Creating nodes
+--------------
+
+To create a node from code, call its ``new()`` method like for any other
+class-based datatype.
+
+You can store the newly created node's reference in a variable and call
+``add_child()`` to add it as a child of the node to which you attached the
+script.
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    var sprite
+
+    func _ready():
+        var sprite = Sprite.new() # Create a new Sprite.
+        add_child(sprite) # Add it as a child of this node.
+
+ .. code-tab:: csharp
+
+    private Sprite _sprite;
+
+    public override void _Ready()
+    {
+        base._Ready();
+
+        _sprite = new Sprite(); // Create a new Sprite.
+        AddChild(_sprite); // Add it as a child of this node.
+    }
+
+To delete a node and free it from memory, you can call its ``queue_free()``
+method. Doing so queues the node for deletion at the end of the current frame
+after it finished processing. At that point, the engine removes the node from
+the scene and frees the object in memory.
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    sprite.queue_free()
+
+ .. code-tab:: csharp
+
+    _sprite.QueueFree();
+
+You can alternatively call ``free()`` to immediately destroy the node. You
+should do this with care as any reference to it will instantly become ``null``.
+We recommend using ``queue_free()`` unless you know what you're doing.
+
+When you free a node, it also frees all its children. Thanks to this, to delete
+an entire branch of the scene tree, you only have to free the topmost parent
+node.
+
+Instancing scenes
+-----------------
+
+Instancing a scene from code happens in two steps:
+
+1. Loading the scene from the hard drive.
+2. Creating an instance of the loaded :ref:`PackedScene <class_PackedScene>`
+   resource.
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    var scene = load("res://MyScene.tscn")
+
+ .. code-tab:: csharp
+
+    var scene = GD.Load<PackedScene>("res://MyScene.tscn");
+
+Preloading it can improve the user's experience as it happens when parsing the
+script. This feature is only available with GDScript.
```

Comment:
Can we detail why it improves the experience here please? 
```suggestion
Preloading the scene can improve the user's experience as the load operation happens when before the script starts and not during scene execution.. This feature is only available with GDScript.
```

Reply:
Sure, thanks